src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -41,8 +41,9 @@ struct FancyZones : public winrt::implements<FancyZones, IFancyZones, IFancyZone
     }
     IFACEMETHODIMP_(GUID) GetCurrentMonitorZoneSetId(HMONITOR monitor) noexcept
     {
-        if (auto it = m_zoneWindowMap.find(monitor); it != m_zoneWindowMap.end()) {
-            return it->second->ActiveZoneSet()->Id();
+        if (auto it = m_zoneWindowMap.find(monitor); it != m_zoneWindowMap.end() && it->second->ActiveZoneSet()) {
+            const auto activeZoneSet = it->second->ActiveZoneSet();
+            return activeZoneSet ? activeZoneSet->Id() : GUID_NULL;

```

Comment:
nullptr validity is already checked with `&& it->second->ActiveZoneSet()` => no need for ternary here.

Reply:
Oh sorry, completely overlooked that I've added the check in if statement.